### DISCUSSION TITLE
Changing SCS from 30 kHz to 15 kHz

### DISCUSSION THREAD

**Discussion by praghur** [srsran/srsRAN_Project#785] [General Help] [answered]
Hello all.

I am using x310 as gNB for my system. I had initially set up the configuration file for a SCS of 30 kHz. However, now when I want to test the system with SCS of 15 kHz, I am having some trouble with it. I changed the parameter to `common_scs: 15`.

After that, I am getting an error
--== srsRAN gNB (commit 4ac5300d4) ==--

Common SCS 15kHz is not equal to SSB SCS 30kHz. Different SCS for common and SSB is not supported.
srsRAN ERROR: Invalid configuration detected.

I found `ssb_scs` under mobility but when I added in the configuration file, it is unable to parse the code. Please see attached gnb configuration code. 
```
mobility: 
  cells:                                      
   - ssb_scs:  15  
```       

--== srsRAN gNB (commit 4ac5300d4) ==--

INI was not able to parse mobility.++
Run with --help for more information.

I am using srsran_project (commit: 4ac5300d4927b5199af69e6bc2e55d061fc33652). Can you please let me know under which section should I add the `ssb_scs`? If there is a better way to do this, I would like to know that too. 

Thanks a lot in advance.                       
[gnb_x310_1.txt](https://github.com/user-attachments/files/16717292/gnb_x310_1.txt)

**Comment by pgawlowicz** [comment, marked as answer]
See Table 5.4.3.3-1 in TS 38.101.
